workflows/rules/aggregate.smk

```diff
@@ -25,13 +25,14 @@ rule aggregate:
         barcodes=BARCODES,
         metadata=METADATA_DIR,
     output:
-        aggregate_profile=get_data_path(input_type="aggregated", tolist=True),
-        cell_counts=CELL_COUNTS,
+        aggregate_profile=get_data_path(input_type="aggregated"),
+        cell_counts=get_data_path(input_type="cell_counts"),
     log:
-        "logs/aggregate_{file_name}.log",
+        "logs/aggregate_{basename}.log",
```

Comment:
just confirming `basename` is defined elsewhere

Reply:
yes, it is defined in the common.smk file